doc/user/content/sql/drop-index.md

```diff
@@ -65,8 +73,26 @@ SHOW INDEXES FROM q01;
 | materialize.public.q01 | materialize.public.q01_geo_idx | ...
 +------------------------+--------------------------------+-----
 ```
+
+You can use the unqualified index name (`q01_geo_idx`) rather the fully qualified name (`materialize.public.q01_geo_idx`).
+
+
```

Comment:
Looking at some of the other examples, maybe you mean to make this an exhaustive set of commands that perform these actions? I think that could be fine but needs more context that this is what you're showing using some kind of lead in and maybe an unordered list to show parallel construction? I think that this is slightly overwrought, though, and you'd be just fine with only the vanilla command.

Reply:
Jessica asked for examples of CASCADE and RESTRICT. Since Markdown doesn't play nice with codeblocks and unordered lists, and codephrases aren't consistent with what we do elsewhere, I've add an explanatory note.

Reply:
Markdown does play nicely with code blocks and unordered lists. If you can't get something to work, you should ask for help.